File: Examples.GridDesktop/CSharp/GridDesktop.Examples/WorkingWithCells/UsingFormatPainter.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace GridDesktop.Examples.WorkingWithCells
{
    public partial class UsingFormatPainter : Form
    {
        public UsingFormatPainter()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            // ExStart:UseOnce
            // Starting format painter to paint once
            gridDesktop1.StartFormatPainter(false);
            // ExEnd:UseOnce
        }

        private void button2_Click(object sender, EventArgs e)
        {
            // ExStart:UseAlways
            // Starting format painter to keep painting forever
            gridDesktop1.StartFormatPainter(true);
            // ExEnd:UseAlways
        }

        private void button3_Click(object sender, EventArgs e)
        {
            // ExStart:EndUse
            // Ending format painter to stop painting
            gridDesktop1.EndFormatPainter();
            // ExEnd:EndUse
        }
    }
}

File: Examples.GridDesktop/VisualBasic/GridDesktop.Examples/WorkingWithCells/UsingFormatPainter.vb
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace WorkingWithCells
	Public Partial Class UsingFormatPainter
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(sender As Object, e As EventArgs)
			' ExStart:UseOnce
			' Starting format painter to paint once
			gridDesktop1.StartFormatPainter(False)
			' ExEnd:UseOnce
		End Sub

		Private Sub button2_Click(sender As Object, e As EventArgs)
			' ExStart:UseAlways
			' Starting format painter to keep painting forever
			gridDesktop1.StartFormatPainter(True)
			' ExEnd:UseAlways
		End Sub

		Private Sub button3_Click(sender As Object, e As EventArgs)
			' ExStart:EndUse
			' Ending format painter to stop painting
			gridDesktop1.EndFormatPainter()
			' ExEnd:EndUse
		End Sub
	End Class
End Namespace
